for/wordpress.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: page
+title: Codeception for WordPress
+hero: wordpress_hero.html
+sidebar: |
+
+  ## Codeception Tests
+
+  * Combine **all testing levels** (acceptance, functional, unit)
+  * use WordPress defined functions, constants, classes and methods in any test
+  * can be written in **BDD** format with Gherkin
+
+  ## Reference
+
+  * [WPBrowser modules library](https://github.com/lucatume/wp-browser) 
+
+---
+
+## Install
+
+Install latest stable Codeception via Composer:
+
+```bash
+composer require codeception/codeception --dev
+```
+
+Install latest stable WPBrowser package via Composer:
+
+```bash
+composer require lucatume/wp-browser --dev
+```
+
+## Setup
+
+To fully use the WordPress specific modules of the WPBrowser suite you need to setup a local WordPress installation; this is in no way different from what's required to locally develop a WordPress theme or plugin.  
+In the examples below I assume that the local WordPress installation root directory is `/var/www/wordpress`, that it responds at the `http://wordpress.dev` URL and that I am developing a plugin called "Acme Plugin".  
+While it's possible to start from a default Codeception configuration the WPBrowser package comes with its own CLI tool (`wpcept`) and its own bootstrap command.  
+Scaffold the tests using the latter
+
+```bash
+wpcept bootstrap
+```
+
+The command creates the `tests` directory and scaffold the `acceptance`, `functional`, `integration` and `unit` suites creating a `tests` sub-directory and a configuration file for each.  
+Edit the `codeception.yml` file to match your local WordPress setup and point the modules to the local WordPress installation folder, the local WordPress URL and so on.  
+If you are using the `WPLoader` module in your tests take care to create a dedicated database for it and not to use the same database the `Db` or `WPDb` modules might use.  
+The use of the modules defined in the WPBrowser package is not tied to this bootstrap though so feel free to set up Codeception in any other way.
+
+## Integration Tests
+Commonly "WordPress unit tests" are not related to classical unit tests but to integration tests. The difference is that unit tests are supposed to test a class methods in complete isolation, while integration tests check how components work inside WordPress. That's why, to prepare WordPress for testing, you should enable `WPLoader` module into `integration.suite.yml`.  
+The `WPLoader` module: it takes care of loading, installing and configuring a fresh WordPress installation before each test method runs.  
+To handle the heavy lifting the module requires some information about the local WordPress installation: in the `codeception.yml` file configure it to match your local setup; with reference to the example above the module configuration might be:
+
+```yaml
+modules:
+    enabled:
```

Comment:
I'm not quite sure but inside `codeception.yml` it is better to pass configuration as

```
modules:
    config:
```